### ISSUE TITLE
[Item][StormPeaks] Everfrost Chip

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4031]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Spawned in wrong phase. The chips can be seen in preHodir phase before you unlock the dailies or faction.
**Expected behaviour**: Tell us what should happen instead
After you complete the chain to unlock the Sons of Hodir. They should randomly spawn. The first one you collect should offer a quest.
**Steps to reproduce the problem**:

1. Go to zone with out doing hodir line you can find one around 69,63 it can be looted but empty.
2. Complete the hodie quest line to phase the are and you wont see any.
3. You can complete the quest line to unlock the faction and never see them however if you fly away just enough to get away from the lake you can see them again due to phasing.

**Include proofs for this behaviour**
No Chip
![wowscrnshot_122516_160653](https://cloud.githubusercontent.com/assets/12252441/21472896/615894ce-cabc-11e6-8a5d-e091873106cc.jpg)
See chip once moved away.
![wowscrnshot_122516_161155](https://cloud.githubusercontent.com/assets/12252441/21472908/ea89aa3a-cabc-11e6-9583-c0aafd9892f6.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
**Include the ID for the game objects, npcs (creatures,p
game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[Everfrost Chip](http://www.wowhead.com/item=44725/everfrost-chip)
[Quest after looting first one](http://www.wowhead.com/quest=13420/everfrost)
[Repeatable quest after first one. Also map with spawn loactions](http://www.wowhead.com/quest=13421/remember-everfrost)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Gameobject: Update Phasing for Everfrost Chip

* closes https://github.com/dalaranwow/dalaran-wow/issues/4031
* commit: 26e6a6fc496708d33a3e0df6a858172fb6ba0909

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Lothloryen** [comment]
@TeridanDW Could you please check the state of this issue on PTR? The phasemask from the objects was corrected very long time ago.

**Comment by TeridanDW** [comment]
Sure, GO Everfrost Chip seems to be spawned correctly, however the quest item that should drop from them seems to have the wrong conditions as I was never able to loot it.
Conditions:
``` sql
INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
(4,26782,44725,0,0,9,0,13420,0,0,1,0,0,'',NULL),
(4,26782,44725,0,0,8,0,13420,0,0,1,0,0,'',NULL),
(4,26782,44725,0,0,5,0,1119,8,0,0,0,0,'',NULL);
```

**Comment by Jindra555** [comment]
08/06/2018, no items can be looted.
